takealot_importer.user.js

```diff
@@ -629,4 +635,4 @@ Countries["Heard Island and McDonald Islands"] = "HM";
 Countries["Iran, Islamic Republic of"] = "IR";
 Countries["Saint Pierre and Miquelon"] = "PM";
 Countries["Saint Helena"] = "SH";
-Countries["Svalbard and Jan Mayen"] = "SJ";
+Countries["Svalbard and Jan Mayen"] = "SJ";
```

Comment:
This removing of trailing newline is unrelated to the change. Nothing serious, but it should be avoided.